<?xml version="1.0"?>
<launch>
    <arg name="object_name" default="HV8" />
  <arg name="icpRefine" default="False" />
  <arg name="arch" default="PointNet_Segmentation" />
  <arg name="process_swich" default="object_segment" />
  <arg name="resolution" default="8192" />
  <arg name="load_path" default="/home/ericlab/ros_package/denso_ws/src/denso_run/denso_pkgs/pose_estimator_pkg/estimator/load_model/semantic/semantic_demo.pth"/>
  <arg name="child_frame_id" default="estimated_tf" />
  <arg name="header_frame_id" default="photoneo_center_optical_frame" />
  <arg name="dataset_mode" default="semantic_segmentation" />
 
  <arg name="process_swicth2" default="raugh_recognition" />
  <arg name="dataset_mode2" default="pose_estimation" />
  <arg name="arch2" default="PointNet_Pose" />
  <arg name="load_path2" default="/home/ericlab/ros_package/denso_ws/src/denso_run/denso_pkgs/pose_estimator_pkg/estimator/load_model/raugh_recognition/raugh_demo.pth" />
  <arg name="resolution2" default="1024" />

    <node name="estimator_server" pkg="estimator" type="semantic_server.py" output="screen" >
        <param name="object_name" value="$(arg object_name)" />
        <param name="process_swich" value="$(arg process_swich)" />
        <param name="arch" value="$(arg arch)" />
        <param name="resolution" value="$(arg resolution)" />
        <param name="load_path" value="$(arg load_path)"/>
        <param name="child_frame_id" value="$(arg child_frame_id)"/>
        <param name="header_frame_id" value="$(arg header_frame_id)"/>
        <param name="dataset_mode" value="$(arg dataset_mode)"/>

        <param name="process_swicth2" value="$(arg process_swicth2)" />
        <param name="dataset_mode2" value="$(arg dataset_mode2)" />
        <param name="arch2" value="$(arg arch2)" />
        <param name="resolution2" value="$(arg resolution2)" />
        <param name="load_path2" value="$(arg load_path2)"/>
    </node>
</launch>